config/clients/java/template/Java/libraries/native/build.gradle.mustache

```diff
@@ -0,0 +1,106 @@
+
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            artifactId = '{{artifactId}}'
+            from components.java
+        }
+    }
+}
+
+task execute(type:JavaExec) {
+   main = System.getProperty('mainClass')
+   classpath = sourceSets.main.runtimeClasspath
+}
+
+task sourcesJar(type: Jar, dependsOn: classes) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    classifier = 'javadoc'
+    from javadoc.destinationDir
+}
+
+artifacts {
+    archives sourcesJar
+    archives javadocJar
+}
+
+
+ext {
+    {{#swagger1AnnotationLibrary}}
+    swagger_annotations_version = "1.6.9"
+    {{/swagger1AnnotationLibrary}}
+    {{#swagger2AnnotationLibrary}}
+    swagger_annotations_version = "2.2.9"
+    {{/swagger2AnnotationLibrary}}
+    jackson_version = "2.14.1"
+    jakarta_annotation_version = "1.3.5"
+    junit_version = "4.13.2"
+    {{#hasFormParamsInSpec}}
+    httpmime_version = "4.5.13"
+    {{/hasFormParamsInSpec}}
+}
+
+dependencies {
+    {{#swagger1AnnotationLibrary}}
+    implementation "io.swagger:swagger-annotations:$swagger_annotations_version"
+    {{/swagger1AnnotationLibrary}}
+    {{#swagger2AnnotationLibrary}}
+    implementation "io.swagger.core.v3:swagger-annotations:$swagger_annotations_version"
+    {{/swagger2AnnotationLibrary}}
+    implementation "com.google.code.findbugs:jsr305:3.0.2"
+    implementation "com.fasterxml.jackson.core:jackson-core:$jackson_version"
+    implementation "com.fasterxml.jackson.core:jackson-annotations:$jackson_version"
+    implementation "com.fasterxml.jackson.core:jackson-databind:$jackson_version"
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:$jackson_version"
+    implementation "org.openapitools:jackson-databind-nullable:0.2.1"
+    implementation "jakarta.annotation:jakarta.annotation-api:$jakarta_annotation_version"
+    {{#hasFormParamsInSpec}}
+    implementation "org.apache.httpcomponents:httpmime:$httpmime_version"
+    {{/hasFormParamsInSpec}}
+    testImplementation "junit:junit:$junit_version"
+}
+
+// Use spotless plugin to automatically format code, remove unused import, etc
+// To apply changes directly to the file, run `gradlew spotlessApply`
+// Ref: https://github.com/diffplug/spotless/tree/main/plugin-gradle
+spotless {
+    // comment out below to run spotless as part of the `check` task
+    enforceCheck false
+    format 'misc', {
+        // define the files (e.g. '*.gradle', '*.md') to apply `misc` to
+        target '.gitignore'
+        // define the steps to apply to those files
+        trimTrailingWhitespace()
+        indentWithSpaces() // Takes an integer argument if you don't like 4
+        endWithNewline()
+    }
+    java {
+        // don't need to set target, it is inferred from java
+        // apply a specific flavor of google-java-format
+        googleJavaFormat('1.8').aosp().reflowLongStrings()
```

Comment:
Suggest the Palantir Java Formatter as both a more minimalistic implementation, and also producing more readable output:

https://github.com/palantir/palantir-java-format#motivation--examples

```suggestion
        palantirJavaFormat()
```